aws/cloudformation/standalone/marketing/deployment/marketing.yml.erb

```diff
@@ -484,6 +492,13 @@ Resources:
               - 'ecs:DescribeServices'
               - 'ecs:UpdateService'
             Resource: !Ref FargateService
+          - Effect: Allow
+            Action:
+              - "s3:PutObject"
+              - "s3:ListBucket"
+            Resource:
+              - !Sub "arn:aws:s3:::cdo-marketing-static-assets-${Environment}"
```

Comment:
I think we need to update these to by dynamically populated from the Bucket Resource we provisioned below.